src/foam/core/Static.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.CLASS({
+  package: 'foam.core',
+  name: 'Static',
+  extends: 'foam.core.AbstractMethod',
+
+  documentation: 'An Axiom for defining static methods.',
+
+  properties: [
+    //get prop from foam.core.AbstractMethod
+    { name: 'code', required: true },//TODO javaCode
+    { name: 'static', value: true }
+  ],
+
+  methods: [
+    function exportAs(obj) {
+      /*var m = obj[this.name];

```

Comment:
Remove comment.

Reply:
You also need to implement installInClass() and installInProto() to add the static method to the JS class/proto.